src/wechaty/wechaty.py

```diff
@@ -684,6 +684,11 @@ async def scan_listener(payload: EventScanPayload):
                         else payload.qrcode
                     if payload.status == ScanStatus.Waiting:
                         qr_terminal(qr_code)
+                        log.info(
+                            'or you can scan qrcode from: '
+                            'https://wechaty.js.org/qrcode/%s',
+                            qr_code
+                        )
```

Comment:
If the `qrcode` in terminal can't be scanned because of size, you can scan it from website.